jekyll/_layouts/classic-docs.html

```diff
@@ -28,6 +28,7 @@
 	</div><!-- .main-body -->
 	{% include global-footer.html %}
 	<script src="{{ "/assets/js/vendor.min.js" | prepend: site.baseurl }}" type="text/javascript"></script>
+  {% include app.html %}
```

Comment:
I'm not sure in what other files we need to include this. Maybe @trevor-circleci  would know?

Reply:
@JuanCaicedo What's happening here? Where is app.html coming from? 

Note: this may be fixed in later commits.

Reply:
I'm not sure where app.html is coming from. I don't see it in the includes folder. How did you generate this?